### PR TITLE
use bpaf to parse command line options

### LINKED ISSUE #1: コマンドパースにライブラリを使う
[clap](https://docs.rs/clap/latest/clap/)や[bpaf](https://docs.rs/bpaf/latest/bpaf/)など

ref:

- [clapの使い方まとめ - ぽよメモ](https://poyo.hatenablog.jp/entry/2022/10/10/170000)
- [Bpaf : Rustにおける軽量かつ柔軟なCommand Line Argument Parser](https://zenn.dev/kirohi/articles/introduce-bpaf)

### PR DESCRIPTION
**Pull request by pacak**
Fixes #1 :)

### CODE REVIEW

**Comment by mehm8128** [comment]
Thank you for your PR!
Because I'm a beginner of Rust, it will take a few days to check the changes.
Please wait until I look over and understand your PR🙏

**Comment by pacak** [comment]
No rush. I'm happy to answer any questions about the code I wrote if you have any.

File: src/main.rs [hunk 1]
```diff
@@ -4,51 +4,102 @@ mod command;
...
-                command::commit::commit(message.to_string());
-            }
-            _ => {
-                println!("no commit message");
-                return;
+    let args = options();
+    println!("Args: {args:?}");
+
+    match args {
+        Command::Init => command::init::init(),
```
Comment (mehm8128) [inline src/main.rs]: We don't explore `.git` directory when command is `init`, so I want you to execute `init` command checking before `find_git_root`.

**Review by mehm8128 — COMMENTED** [review]
Could you fix only one point?🙏

File: src/main.rs (hunk 1 above)
Comment (pacak) [reply src/main.rs]: With aaba8707dbc1cf61f39ce39e3968beb0afd115d5 it should execute `find_git_root` only if command is not `Command:init`

**Review by mehm8128 — APPROVED** [review]
Thank you!
`bpaf` looks good command line argument parser, so I want to use it in the future!

**Pull request merged**